webpack.config.js

```diff
@@ -6,25 +6,33 @@ const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
 
 module.exports = {
-  entry: glob.sync("./!(static)/static/webpack/**/*?(-)index.js").reduce((obj, el) => {
-    // We include the path inside the webpack folder in the name
-    const relativePath = [];
-    const fullPath = path.parse(el);
-    for (const dir of fullPath.dir.split("/").reverse()) {
-      if (dir === "webpack") {
-        break;
+  entry: glob
+    .sync("./!(static)/static/webpack/**/*?(-)index.[j|t]s?(x)")
+    .reduce((obj, el) => {
+      // We include the path inside the webpack folder in the name
+      const relativePath = [];
+      const fullPath = path.parse(el);
+      for (const dir of fullPath.dir.split("/").reverse()) {
+        if (dir === "webpack") {
+          break;
+        }
+        relativePath.push(dir);
       }
-      relativePath.push(dir);
-    }
-    relativePath.push(fullPath.name);
-    obj[relativePath.join("/")] = `./${el}`;
-    return obj;
-  }, {}),
+      relativePath.push(fullPath.name);
+      obj[relativePath.join("/")] = `./${el}`;
+      return obj;
+    }, {}),
+  cache: {
+    type: "filesystem", // This reduces typescript compilation time like crazy
+  },
```

Comment:
```suggestion
  cache: true,
```
Ca réduit encore plus le temps de compilation